HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -1170,7 +1170,7 @@ settings.advanced.natives_directory=Native Library Path
 settings.advanced.natives_directory.choose=Choose the location of the desired native library
 settings.advanced.natives_directory.custom=Custom
 settings.advanced.natives_directory.default=Default
-settings.advanced.natives_directory.hint=This option is intended only for users of Apple M1 or other not officially supported platforms. Please do not edit this option unless you know what you are doing.\n\
+settings.advanced.natives_directory.hint=This option is intended only for users of Apple Silicon or other not officially supported platforms. Please do not edit this option unless you know what you are doing.\n\
```

Apple 官方和 Wikipedia 的拼写为 `Apple silicon`，而且在 HMCL 语言文件中出现了不止一次，建议跟随官方写法。
```suggestion
settings.advanced.natives_directory.hint=This option is intended only for users of Apple silicon or other not officially supported platforms. Please do not edit this option unless you know what you are doing.\n\
```